dipy/reconst/multi_voxel.py

```diff
@@ -99,6 +100,19 @@ def new_fit(self, data, mask=None, **kwargs):
                 data_to_fit[ii : ii + vox_per_chunk]
                 for ii in range(0, data_to_fit.shape[0], vox_per_chunk)
             ]
+            # FIXME: weights are now broken, DUE TO PULL REQUEST MADE WHILE MINE WAS WAITING
+            #        I was passing weights as a kwarg to make it work,
+            #        I don't think it will be possible to do in the way they have set-up here
+            #        thanks, DiPy team, you really do go above and beyond to make things difficult
+            print("in multi_voxel here are the keywords")
+            print(kwargs)
+            print(weights_is_array)
+            # FIXME: despite "return_leverages: True" being in kwargs, mvf does not have leverages,
+            #        so something is broken AND I AM BEING FORCED TO REBASE ON THIS BROKEN CODE
+            #        I have tested, and return_leverages is coming out as false, so clearly
+            #        there is a bug here in the code THAT I AM BEING FORCE TO REBASE ON
+            if weights_is_array:
+                raise ValueError("Cannot handle custom weights here at the moment")
```

Comment:
@arokem, Can you look at this please ? it is about your last PR `paramap` and parallelization of model fitting. Thank you !

Reply:
related to [#2593](https://github.com/dipy/dipy/pull/2593)

Reply:
Sorry about this: I am not exactly sure what the issue is. Is this simply because kwargs are not getting passed here? https://github.com/dipy/dipy/blob/master/dipy/reconst/multi_voxel.py#L60

Reply:
For what it's worth, we are trying to make things as easy as we can, but we have limited bandwidth and this stuff is complicated, so we ask for a little bit of grace from developers who want to contribute to the project. We really want to support you in your work here, but we are only humans and there are only 24 hours per day. So, please be patient with us. No one is intentionally trying to make this hard on you.

Reply:
And I ask in return for grace when I get frustrated, since I am only human and have even less than 24 hours a day. Nobody is trying to make it hard, but they are making it hard nonetheless, for reasons that are controllable. So, I suggest reflection on my point of view, not rejection of it, please.